spec/menu_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'menu'
+
+describe Menu do
+  subject (:menu) { described_class.new(dishes) }
```

Comment:
Does this work this way round? Like putting dishes in there before you create it in the next line? Genuine question cause I don't know